README.md

```diff
@@ -630,6 +630,51 @@ Otherwise the client will raise a `HTTPError`:
 </table>
 </details>
 
+## Get all templates
+
+```python
+response = notifications_client.get_all_templates(
+    template_type=None # optional - SMS_TYPE | EMAIL_TYPE
```

Comment:
we should reference the valid types here in this doc - think this should say `# optional - see https://github.com/alphagov/notifications-python-client/blob/master/README.md#template_type` or something. or actually, if you include that outside of the code block, then it can be a nice hyperlink